src/translation/translation_ko_KR.ts

```diff
@@ -0,0 +1,4164 @@
+        <source>Enable Small Network Buffers</source>
+        <translation>소규모 네트워크 버퍼 활성화</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="608"/>
+        <source>Buffer Delay</source>
+        <translation>버퍼 지연</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="103"/>
+        <source>Country/Region</source>
+        <translation>국가/지역</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="620"/>
+        <source>(preferred)</source>
+        <translation>(선호)</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="633"/>
+        <source>(default)</source>
+        <translation>(기본)</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="646"/>
+        <source>(safe)</source>
+        <translation>(안전)</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="380"/>
+        <source>Driver Setup</source>
+        <translation>드라이버 설정</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="40"/>
+        <source>My Profile</source>
+        <translation>내 프로필</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="79"/>
+        <source>Musician&apos;s Profile</source>
+        <translation>뮤지션 프로필</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="89"/>
+        <source>Alias/Name</source>
+        <translation>별칭/이름</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="96"/>
+        <source>Instrument</source>
+        <translation>악기</translation>
+    </message>
+    <message>
+        <source>Country</source>
+        <translation type="vanished">국가</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="110"/>
+        <source>City</source>
+        <translation>도시</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="117"/>
+        <source>Skill</source>
+        <translation>스킬</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="210"/>
+        <source>User Interface</source>
+        <translation>사용자 인터페이스</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="227"/>
+        <source>Meter Style</source>
+        <translation>미터 스타일</translation>
+    </message>
+    <message>
+        <location filename="../clientsettingsdlgbase.ui" line="241"/>
+        <source>Mixer Rows</source>
+        <translation믹서 줄</translation>
```

Comment:
There's a ">" character missing after "translation".

Are you directly editing the .ts file? These errors are easily avoided if you use QtLinguist (or any other editor that can handle .ts files) instead. The problem is, if there are more of these syntax errors, this file might not load properly in QtLinguist, so no point switching now...

Reply:
yes, I do.
I understand it, Thank you.